tests/logictest/suites/mode/cluster/04_0002_explain_v2

```diff
@@ -116,7 +116,7 @@ EvalScalar
 ├── scalars: [t1.a (#0), t1.b (#1), t2.a (#2), t2.b (#3)]
 └── LogicalJoin: INNER
     ├── equi conditions: [(t1.a (#0) = t2.a (#2)) AND (t1.b (#1) = t2.b (#3))]
-    ├── other conditions: [t1.a (#0) > 2]
```

Comment:
Should this predicate be pushed down to `t1`? It's disappeared in the explain result.